tasks/docker-rootless.yml

```diff
@@ -0,0 +1,60 @@
+---
+- name: Ensure dockerd-rootless-setup.sh is installed
+  package:
+    name:
+      - uidmap
+      - docker-ce-rootless-extras
+    state: present
+  when: ansible_distribution != "CentOS"
+
+- name: Ensure dockerd-rootless-setup.sh is installed
+  package:
+    name:
+      - shadow-utils
+      - docker-ce-rootless-extras
+    state: present
+  when: ansible_distribution == "CentOS"
+
+- name: Stop any running root instances of docker daemon
```

Comment:
It may just be me but going completely rootless caused quite a lot of issues with permissions, mapping multiple users/uids (like root AND www-data for example), etc. So I would like stopping/removing the root docker daemon to be optional to be able to run Docker in a "dual setup".

Reply:
This is a good idea, I'd be happy to try it but I didn't think it was possible.  When I run the `dockerd-rootless-setuptool.sh` script to install, I get an error if the docker.sock file exists, even if it isn't listening, which is why I have to explicitly remove it in the following step.  

There is the `--force` option, but I don't know how safe that is to use.

It looks like that's what your script does in #412.  What flavor of Linux did you use that script on?

Reply:
I ran it under Ubuntu Server 22.04 LTS. Just ran it again and can confirm that it indeed doesn't work. Probably added the condition after the fact, sorry for the confusion 😁

Reply:
No problem.  It's too bad it doesn't work, it would be nice to have an option.  Not sure how it could be enforced though - I believe users still need to be in the docker group to run rootless docker, so if root docker is still available to them, it kind of defeats the purpose of locking down the host IMO.

Reply:
I was able to run rootless Docker without my user being in the "docker" group.